### ISSUE TITLE
`cursive.repl.runtime.errors/ex->data` throws

### ISSUE THREAD

**Issue by onetom** [cursive-ide/cursive#2993]
# Reproduction

It requires this lib:
```edn
functionalbytes/redelay
  {:mvn/version "2.0.0"
   :changelog   "https://github.com/aroemers/redelay/blob/master/CHANGELOG.md"}
```

When I run
```clojure
(cursive.repl.runtime.errors/ex->data
  (ex-info "x" {:r (redelay.core/state 1)}) 
  :execution)
```

it throws
```
Execution error (AbstractMethodError) at redelay.core.State/empty (core.clj:-1).
Method redelay/core/State.empty()Lclojure/lang/IPersistentCollection; is abstract
java.lang.AbstractMethodError: Method redelay/core/State.empty()Lclojure/lang/IPersistentCollection; is abstract
             (Unknown Source) redelay.core/redelay.core.State
                core.clj:5300 clojure.core/empty
                  walk.clj:50 clojure.walk/walk
                  walk.clj:53 clojure.walk/postwalk
                  walk.clj:53 clojure.walk/postwalk
                core.clj:2641 clojure.core/partial[fn]
                  walk.clj:46 clojure.walk/walk
                  walk.clj:53 clojure.walk/postwalk
                  walk.clj:53 clojure.walk/postwalk
                core.clj:2641 clojure.core/partial[fn]
                core.clj:2772 clojure.core/map[fn]
              LazySeq.java:50 clojure.lang.LazySeq.force
              LazySeq.java:89 clojure.lang.LazySeq.realize
             LazySeq.java:106 clojure.lang.LazySeq.seq
                  RT.java:555 clojure.lang.RT.seq
                 core.clj:139 clojure.core/seq
             protocols.clj:24 clojure.core.protocols/seq-reduce
             protocols.clj:74 clojure.core.protocols/fn
             protocols.clj:74 clojure.core.protocols/fn
             protocols.clj:13 clojure.core.protocols/fn[fn]
                core.clj:6965 clojure.core/reduce
                core.clj:7038 clojure.core/into
                  walk.clj:50 clojure.walk/walk
                  walk.clj:53 clojure.walk/postwalk
                  walk.clj:53 clojure.walk/postwalk
                core.clj:2641 clojure.core/partial[fn]
                  walk.clj:46 clojure.walk/walk
                  walk.clj:53 clojure.walk/postwalk
                  walk.clj:53 clojure.walk/postwalk
                core.clj:2641 clojure.core/partial[fn]
                core.clj:2772 clojure.core/map[fn]
              LazySeq.java:50 clojure.lang.LazySeq.force
              LazySeq.java:89 clojure.lang.LazySeq.realize
             LazySeq.java:106 clojure.lang.LazySeq.seq
                 Cons.java:41 clojure.lang.Cons.next
                  RT.java:733 clojure.lang.RT.next
                  core.clj:64 clojure.core/next
            protocols.clj:168 clojure.core.protocols/fn
            protocols.clj:123 clojure.core.protocols/fn
             protocols.clj:19 clojure.core.protocols/fn[fn]
             protocols.clj:31 clojure.core.protocols/seq-reduce
             protocols.clj:74 clojure.core.protocols/fn
             protocols.clj:74 clojure.core.protocols/fn
             protocols.clj:13 clojure.core.protocols/fn[fn]
                core.clj:6965 clojure.core/reduce
                core.clj:7038 clojure.core/into
                  walk.clj:50 clojure.walk/walk
                  walk.clj:53 clojure.walk/postwalk
                  walk.clj:53 clojure.walk/postwalk
                core.clj:2641 clojure.core/partial[fn]
                core.clj:2770 clojure.core/map[fn]
              LazySeq.java:50 clojure.lang.LazySeq.force
              LazySeq.java:89 clojure.lang.LazySeq.realize
             LazySeq.java:106 clojure.lang.LazySeq.seq
                  RT.java:555 clojure.lang.RT.seq
                 core.clj:139 clojure.core/seq
             protocols.clj:24 clojure.core.protocols/seq-reduce
             protocols.clj:74 clojure.core.protocols/fn
             protocols.clj:74 clojure.core.protocols/fn
             protocols.clj:13 clojure.core.protocols/fn[fn]
                core.clj:6965 clojure.core/reduce
                core.clj:7038 clojure.core/into
                  walk.clj:50 clojure.walk/walk
                  walk.clj:53 clojure.walk/postwalk
                  walk.clj:53 clojure.walk/postwalk
                core.clj:2641 clojure.core/partial[fn]
                  walk.clj:46 clojure.walk/walk
                  walk.clj:53 clojure.walk/postwalk
                  walk.clj:53 clojure.walk/postwalk
                core.clj:2641 clojure.core/partial[fn]
                core.clj:2772 clojure.core/map[fn]
              LazySeq.java:50 clojure.lang.LazySeq.force
              LazySeq.java:89 clojure.lang.LazySeq.realize
             LazySeq.java:106 clojure.lang.LazySeq.seq
                  RT.java:555 clojure.lang.RT.seq
                 core.clj:139 clojure.core/seq
             protocols.clj:24 clojure.core.protocols/seq-reduce
             protocols.clj:74 clojure.core.protocols/fn
             protocols.clj:74 clojure.core.protocols/fn
             protocols.clj:13 clojure.core.protocols/fn[fn]
                core.clj:6965 clojure.core/reduce
                core.clj:7038 clojure.core/into
                  walk.clj:50 clojure.walk/walk
                  walk.clj:53 clojure.walk/postwalk
                  walk.clj:53 clojure.walk/postwalk
                errors.clj:36 cursive.repl.runtime.errors/ex->data
                errors.clj:28 cursive.repl.runtime.errors/ex->data
             NO_SOURCE_FILE:1 user/eval27442
             NO_SOURCE_FILE:1 user/eval27442
           Compiler.java:7691 clojure.lang.Compiler.eval
   interruptible_eval.clj:106 nrepl.middleware.interruptible-eval/evaluator[fn]
   interruptible_eval.clj:101 nrepl.middleware.interruptible-eval/evaluator[fn]
              session.clj:229 nrepl.middleware.session/session-exec[fn]
        SessionThread.java:21 nrepl.SessionThread.run
```

**Comment by cursive-ide** [comment]
Fix released in 1.14.1-eap1 on 2024-12-11.

**Comment by cursive-ide** [comment]
Fix released in 2025.1 on 2025-04-16.
